src/libsystemd/sd-journal/journal-internal.h

```diff
@@ -87,6 +87,7 @@ struct sd_journal {
         uint64_t current_field;
 
         Match *level0, *level1, *level2;
+        Set *exclude;
```

Comment:
I'd like to make the name more descriptive, e.g. `exclude_syslog_identifiers` or so.

Reply:
Refactoring done